### ISSUE TITLE
Make note of outliers in some of the dataframes

### ISSUE THREAD

**Issue by cklaassen** [bandsquad/google-trends-fetch#76]
Mark outliers based on the optimized iqr.py function

**Comment by nathanparekh** [comment]
Outliers in what? 
We should think practically about the end goal. Are we doing day-over-day percent change for each brand and then comparing the percent change?
Remember, none of the brands can be compared to each other anymore.

**Comment by cklaassen** [comment]
What I was thinking is a outlier between itself:
first of all, comparing to percent change isn't the best option in my opinion.  The data google gives us are already percents, so we would be taking a percent of a percent, which doesn't make sense.

two, what we can do is compare the brands using the interquartile range method in itself.  The algorithm I created drowns out the noise and only is interested in when the brand breaks a certain threshold.  

So to sum up, 
We should be comparing straight to the data frame, and comparing to the singular data is fine due to the way my algorithm works.  Tell me if you disagree with this method

**Comment by nathanparekh** [comment]
> outlier between itself
is that a thing?

**Comment by cklaassen** [comment]
This might be easier to understand if I show you with a graph, let me throw together a program to show you
